opm/simulators/utils/DamarisKeywords.hpp

```diff
@@ -104,6 +112,30 @@ getDamarisKeywords(const Parallel::Communication& comm, const std::string& Outpu
     return settings.getKeywords(comm, OutputDir);
 }
 
+template<class TypeTag>
+std::unordered_set<std::string>
+getSetOfIncludedVariables(void) 
+{
+    std::unordered_set<std::string> resuset ;
+    std::string tstr;
+    // The --damaris-limit-variables command line option (defaults to empty string)
+    std::string damarisLimitVars = Parameters::get<TypeTag, Properties::DamarisLimitVariables>();
+    std::stringstream ss(damarisLimitVars); 
+
+    // Use while loop to check the getline() function condition.  
+    while (std::getline(ss, tstr, ',')) {
+        //remove whitespace
+        // tstr.erase(std::remove_if(tstr.begin(), tstr.end(), ::isspace), tstr.end());
```

Comment:
Why did you change this to not use ::isspace?
If keeping current version, delete the commented out line.

Reply:
It did not work/compile - I forget the issue exactly.